[2025-03-01T08:25:11.712+0000] {processor.py:186} INFO - Started process (PID=96) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:11.715+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:25:11.722+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:25:11.721+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:11.810+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:25:11.792+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:25:11.814+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:11.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.197 seconds
[2025-03-01T08:25:42.123+0000] {processor.py:186} INFO - Started process (PID=162) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:42.124+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:25:42.127+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:25:42.127+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:42.143+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:25:42.140+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:25:42.144+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:25:42.167+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.052 seconds
[2025-03-01T08:26:12.936+0000] {processor.py:186} INFO - Started process (PID=228) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:12.938+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:26:12.942+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:26:12.942+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:12.970+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:26:12.962+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:26:12.972+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:13.008+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.084 seconds
[2025-03-01T08:26:43.300+0000] {processor.py:186} INFO - Started process (PID=294) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:43.301+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:26:43.303+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:26:43.303+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:43.318+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:26:43.315+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:26:43.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:26:43.337+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.043 seconds
[2025-03-01T08:27:14.020+0000] {processor.py:186} INFO - Started process (PID=360) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:14.021+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:27:14.026+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:27:14.025+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:14.047+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:27:14.044+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:27:14.048+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:14.080+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.065 seconds
[2025-03-01T08:27:44.358+0000] {processor.py:186} INFO - Started process (PID=426) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:44.359+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:27:44.362+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:27:44.362+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:44.378+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:27:44.375+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:27:44.379+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:27:44.412+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.062 seconds
[2025-03-01T08:28:14.985+0000] {processor.py:186} INFO - Started process (PID=492) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:14.988+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:28:14.993+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:28:14.992+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:15.013+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:28:15.009+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:28:15.014+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:15.036+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.069 seconds
[2025-03-01T08:28:45.187+0000] {processor.py:186} INFO - Started process (PID=558) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:45.188+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:28:45.190+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:28:45.190+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:45.203+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:28:45.200+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:28:45.204+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:28:45.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.054 seconds
[2025-03-01T08:29:15.471+0000] {processor.py:186} INFO - Started process (PID=624) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:15.472+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:29:15.474+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:29:15.474+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:15.487+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:29:15.484+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:29:15.488+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:15.518+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.053 seconds
[2025-03-01T08:29:46.219+0000] {processor.py:186} INFO - Started process (PID=690) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:46.220+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:29:46.223+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:29:46.222+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:46.240+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:29:46.237+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:29:46.241+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:29:46.269+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.057 seconds
[2025-03-01T08:30:17.074+0000] {processor.py:186} INFO - Started process (PID=756) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:17.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:30:17.077+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:30:17.077+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:17.094+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:30:17.090+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:30:17.095+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:17.112+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.044 seconds
[2025-03-01T08:30:47.279+0000] {processor.py:186} INFO - Started process (PID=822) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:47.280+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:30:47.283+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:30:47.283+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:47.303+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:30:47.299+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:30:47.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:30:47.339+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.067 seconds
[2025-03-01T08:31:17.504+0000] {processor.py:186} INFO - Started process (PID=888) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:17.505+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:31:17.508+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:31:17.508+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:17.522+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:31:17.520+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:31:17.524+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:17.554+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.059 seconds
[2025-03-01T08:31:47.786+0000] {processor.py:186} INFO - Started process (PID=954) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:47.787+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:31:47.790+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:31:47.790+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:47.805+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:31:47.802+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:31:47.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:31:47.842+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.062 seconds
[2025-03-01T08:32:18.056+0000] {processor.py:186} INFO - Started process (PID=1020) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:32:18.057+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:32:18.059+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:32:18.059+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:32:18.074+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:32:18.071+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/model_training_multiple_predictions_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/model_training_multiple_predictions_dag.py", line 6, in <module>
    from sklearn.model_selection import train_test_split
ModuleNotFoundError: No module named 'sklearn'
[2025-03-01T08:32:18.075+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:32:18.094+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.044 seconds
[2025-03-01T08:47:31.685+0000] {processor.py:186} INFO - Started process (PID=103) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:47:31.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:47:31.698+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:31.694+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:47:34.210+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:47:34.682+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:34.681+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:model_training_multiple_predictions_dag
[2025-03-01T08:47:34.807+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:34.806+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:model_training_multiple_predictions_dag
[2025-03-01T08:47:35.045+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:35.045+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:model_training_multiple_predictions_dag
[2025-03-01T08:47:35.682+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:35.681+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:model_training_multiple_predictions_dag
[2025-03-01T08:47:36.277+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:36.276+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:model_training_multiple_predictions_dag
[2025-03-01T08:47:36.778+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:36.778+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:model_training_multiple_predictions_dag
[2025-03-01T08:47:37.031+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:37.031+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:model_training_multiple_predictions_dag
[2025-03-01T08:47:37.033+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:37.032+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:47:37.074+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:37.073+0000] {dag.py:3262} INFO - Creating ORM DAG for model_training_multiple_predictions_dag
[2025-03-01T08:47:37.449+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:47:37.449+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:47:38.070+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 6.420 seconds
[2025-03-01T08:48:08.450+0000] {processor.py:186} INFO - Started process (PID=186) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:08.451+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:48:08.454+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:08.454+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:10.378+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:10.741+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:10.740+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:48:10.783+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:10.783+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:48:10.819+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 2.376 seconds
[2025-03-01T08:48:41.000+0000] {processor.py:186} INFO - Started process (PID=264) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:41.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:48:41.005+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:41.004+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:41.661+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:48:41.681+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:41.680+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:48:41.897+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:48:41.896+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:48:41.923+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.931 seconds
[2025-03-01T08:49:12.667+0000] {processor.py:186} INFO - Started process (PID=339) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:12.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:49:12.672+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:12.672+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:14.295+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:14.325+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:14.324+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:49:14.573+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:14.572+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:49:14.599+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.945 seconds
[2025-03-01T08:49:44.845+0000] {processor.py:186} INFO - Started process (PID=413) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:44.846+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:49:44.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:44.849+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:45.573+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:49:45.607+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:45.606+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:49:45.840+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:49:45.840+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:49:45.868+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.031 seconds
[2025-03-01T08:54:34.454+0000] {processor.py:186} INFO - Started process (PID=89) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:54:34.456+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:54:34.459+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:54:34.459+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:54:35.477+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:54:35.507+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:54:35.507+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:54:35.536+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:54:35.536+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:54:35.563+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.119 seconds
[2025-03-01T08:55:05.684+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:05.686+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:55:05.698+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:05.696+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:07.175+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:07.211+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:07.210+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:55:07.543+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:07.543+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:55:07.572+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.910 seconds
[2025-03-01T08:55:38.626+0000] {processor.py:186} INFO - Started process (PID=244) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:38.630+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:55:38.638+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:38.637+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:41.957+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:55:42.041+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:42.040+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:55:42.801+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:55:42.800+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:55:42.877+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 4.274 seconds
[2025-03-01T08:56:13.204+0000] {processor.py:186} INFO - Started process (PID=321) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:13.206+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:56:13.208+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:13.208+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:13.977+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:13.998+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:13.997+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:56:14.170+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:14.170+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:56:14.204+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.007 seconds
[2025-03-01T08:56:44.375+0000] {processor.py:186} INFO - Started process (PID=394) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:44.376+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:56:44.378+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:44.378+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:45.019+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:56:45.041+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:45.041+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:56:45.066+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:56:45.066+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:56:45.094+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.725 seconds
[2025-03-01T08:57:16.057+0000] {processor.py:186} INFO - Started process (PID=469) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:16.059+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:57:16.063+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:16.062+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:17.030+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:17.069+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:17.069+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:57:17.103+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:17.102+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:57:17.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.079 seconds
[2025-03-01T08:57:47.255+0000] {processor.py:186} INFO - Started process (PID=546) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:47.257+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:57:47.262+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:47.261+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:48.408+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:57:48.433+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:48.432+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:57:48.467+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:57:48.467+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:57:48.500+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.256 seconds
[2025-03-01T08:58:19.187+0000] {processor.py:186} INFO - Started process (PID=624) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:19.188+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:58:19.191+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:19.190+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:20.235+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:20.256+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:20.256+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:58:20.279+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:20.278+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:58:20.312+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.131 seconds
[2025-03-01T08:58:50.751+0000] {processor.py:186} INFO - Started process (PID=699) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:50.752+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:58:50.754+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:50.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:51.416+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:58:51.600+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:51.600+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:58:51.623+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:58:51.623+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:58:51.642+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.897 seconds
[2025-03-01T08:59:22.522+0000] {processor.py:186} INFO - Started process (PID=772) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:22.523+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:59:22.527+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:22.527+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:23.218+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:23.415+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:23.414+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:59:23.451+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:23.451+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:59:23.481+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 0.969 seconds
[2025-03-01T08:59:54.274+0000] {processor.py:186} INFO - Started process (PID=852) to work on /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:54.276+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/model_training_multiple_predictions_dag.py for tasks to queue
[2025-03-01T08:59:54.280+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:54.279+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:55.867+0000] {processor.py:925} INFO - DAG(s) 'model_training_multiple_predictions_dag' retrieved from /opt/airflow/dags/model_training_multiple_predictions_dag.py
[2025-03-01T08:59:56.127+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:56.126+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-01T08:59:56.152+0000] {logging_mixin.py:190} INFO - [2025-03-01T08:59:56.152+0000] {dag.py:4180} INFO - Setting next_dagrun for model_training_multiple_predictions_dag to 2024-11-13 00:00:00+00:00, run_after=2025-01-12 00:00:00+00:00
[2025-03-01T08:59:56.183+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/model_training_multiple_predictions_dag.py took 1.922 seconds
